server.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+const errorHandler = (err, req, res, next) => {
+  console.log(err);
+  res.status(500).send(err.message);
+};
+
+app.route("/ping").get((req, res, next) => {
```

Comment:
Consider using a more descriptive route than '/ping'. It would be more meaningful to have a route that reflects the functionality it provides. [medium]